### DISCUSSION TITLE
Search discussions by label using logical OR

### DISCUSSION THREAD

**Discussion by sbvegan** [community/community#72751] [Discussions]
### Select Topic Area

Product Feedback

### Body

Similar to #4507, it would be nice to be able to search discussion labels with a logical OR.

**Comment by Talhadmr** [comment]
I agree with you on this matter. A great potential is being wasted here.

**Comment by alambike** [comment]
Indeed, the [discussion search engine](https://docs.github.com/en/search-github/searching-on-github/searching-discussions) is very limited with respect to label searching for issues and pull requests:

![image](https://github.com/community/community/assets/1025564/fa2cfbec-d0ff-4abd-9c8c-09e0999a6f36)

Any estimate on whether negations and logical OR will ever be available for discussions label searches?
